cmd/abci-cli/abci-cli.go

```diff
@@ -295,6 +310,46 @@ func cmdCommit(c *cli.Context) error {
 	return nil
 }
 
+// Delineate beginning of block with header and hash
+func cmdBeginBlock(c *cli.Context) error {
+	args := c.Args()
+	if len(args) > 2 {
+		return errors.New("Command begin block takes max 2 arguments")
+	}
+	headerBytes, err := stringOrHexToBytes(c.Args()[0])
+	if err != nil {
+		return err
+	}
+	res := client.BeginBlockSync(headerBytes)
```

Comment:
BeginBlock takes two args, a hash and a header: https://github.com/tendermint/abci/blob/master/client/client.go#L40